### PR TITLE
Add list of accepted RFCs to docs

### LINKED ISSUE #1583: Gather all accepted RFC's in one section ?
Since the topic of the RFC template came up at the start of the meeting, I think it would be better to gather up all the accepted RFC's in one section. 

Option 1 : List all of the accepted RFC's with links in [this section](https://mitiq.readthedocs.io/en/stable/contributing.html#proposing-a-new-feature-to-mitiq) of the docs where the template exists. 

Option 2 : Wiki page 

Feel free to add another option as well.

### PR DESCRIPTION
**Pull request by nathanshammah**
## Description
Close #1583.

---

### License

- [x] I license this contribution under the terms of the GNU GPL, version 3 and grant Unitary Fund the right to provide additional permissions as described in section 7 of the GNU GPL, version 3.

Before opening the PR, please ensure you have completed the following where appropriate.
- [ ] I added unit tests for new code.
- [x] I used [type hints](https://www.python.org/dev/peps/pep-0484/) in function signatures.
- [ ] I used [Google-style](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) docstrings for functions.
- [x] I [updated the documentation](../blob/master/docs/CONTRIBUTING_DOCS.md) where relevant.
- [x] Added myself / the copyright holder to the AUTHORS file

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -164,6 +164,25 @@ and add a link to your proposal document (make sure to enable commenting!). For
 weren't able to complete please mention that in the PR description and/or discuss with us on
 [discord](http://discord.unitary.fund).
 
+### List of accepted RFCs
```
Comment (purva-thakre) [inline CONTRIBUTING.md:167]: Should this section be linked in the paragraph above ? The link directs to closed issues on Gihtub. I think the early implementation of the RFC was as an issue when we were still using markdown.

```
Take a look at [previous RFCs](#list-of-accepted-rfcs) for examples on how to fill out your proposal.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -164,6 +164,25 @@ and add a link to your proposal document (make sure to enable commenting!). For
...
+
+This is a list of accepted request-for-comments (RFC) documents by date of creation (reverse chronological order):
+
+- [Implementation of Mitiq calibration RFC](https://docs.google.com/document/d/1EZUJyEEUQUH33UOgSIzCCvXyxP0WLOQn11W0x4Ox4nY/edit) by Andrea Mari (@andreamari) Nov 2, 2022
+
+- [Calibration RFC](https://docs.google.com/document/d/1otUHnTlyNS-0rxGAxltHLF1iD5C9qT9oEZ3jn8VHWgw/edit) by Andrea Mari (@andreamari) Oct 6, 2022
```
Comment (andreamari) [inline CONTRIBUTING.md:173]:
```suggestion
- [Calibration tools for error mitigation RFC (abstract general solutions)](https://docs.google.com/document/d/1otUHnTlyNS-0rxGAxltHLF1iD5C9qT9oEZ3jn8VHWgw/edit) by Andrea Mari (@andreamari) Oct 6, 2022
```
Closer to actual title. Mentioning about abstract high-level solutions.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -164,6 +164,25 @@ and add a link to your proposal document (make sure to enable commenting!). For
 weren't able to complete please mention that in the PR description and/or discuss with us on
 [discord](http://discord.unitary.fund).
 
+### List of accepted RFCs
+
+This is a list of accepted request-for-comments (RFC) documents by date of creation (reverse chronological order):
+
+- [Implementation of Mitiq calibration RFC](https://docs.google.com/document/d/1EZUJyEEUQUH33UOgSIzCCvXyxP0WLOQn11W0x4Ox4nY/edit) by Andrea Mari (@andreamari) Nov 2, 2022
```
[the remark this answers is not in the document]
Comment (natestemen) [reply CONTRIBUTING.md:171]: I like the new title, but no need IMO to include _(Solution 1)_.

**Review by natestemen — APPROVED** [review]
Once Purva and Andrea's comments are addressed, I think this looks great!

File: CONTRIBUTING.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (andreamari) [reply CONTRIBUTING.md:171]: _Solution 1_ removed.

**Review by purva-thakre — APPROVED** [review]
LGTM !

**Pull request merged**